otter/test/utils.py

```diff
@@ -370,6 +370,16 @@ def __init__(self, code, headers, data=None):
         # Data is not part of twisted response object
         self._data = data
 
+    def __eq__(self, other):
+        return (
+            isinstance(other, self.__class__) and
+            self.code == other.code and
+            self.headers == other.headers and
+            self._data == other._data)
+
+    def __neq__(self, other):
+        return self != other
```

Comment:
@lvh:  thanks for the explanation :D  I should have looked this up before I did it - I was just trying to get the test to pass, so pretty sloppy :)